backend/scripts/add_data_base_basic_data.py

```diff
@@ -0,0 +1,33 @@
+import json
```

Comment:
just a simple script to include basic data in the database (now only adding the develop user)